instrumentation/httpd/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(opentelemetry-httpd)
+
+find_package(opentelemetry-cpp REQUIRED)
+find_package(Threads REQUIRED)
+find_package(protobuf REQUIRED)
+find_package(gRPC REQUIRED)
+find_package(CURL REQUIRED)
+find_package(Thrift REQUIRED)
+
+add_library(otel_httpd_module SHARED
+  src/otel/mod_otel.cpp
+  src/otel/opentelemetry.cpp
+)
+
+target_compile_options(otel_httpd_module
+  PRIVATE -Wall -Wextra -fpermissive
+)
+
+install(TARGETS otel_httpd_module DESTINATION ".")
+
+set_target_properties(otel_httpd_module PROPERTIES
+  PREFIX ""
+)
+
+target_include_directories(otel_httpd_module
+  PRIVATE
+  ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
+  "/usr/include/apache2"
+  "/usr/include/apr-1.0"
```

Comment:
I think Apache headers should be searched for to omit hardcoding of this search path.

E.g. it should be possible to do `-DCMAKE_PREFIX_PATH="/usr/local/customapachedir"` and CMake will pick up the include paths from it.

A simple example how to to write such a CMake script can be found at https://github.com/seemk/FastHumanDetection/blob/master/modules/FindKinectV2.cmake (unrelated project, just a simple example how a header file can be searched for)